language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/Filter.java

```diff
@@ -16,4 +17,46 @@ public static Filter fromProto(TransactionFilterOuterClass.Filters filters) {
   }
 
   public abstract TransactionFilterOuterClass.Filters toProto();
+
+  // TODO #14537 this is just one bool now.  Is it likely to change?  If not, inline.
+  public static final class Interface {
+    public final boolean includeInterfaceView;
+
+    public Interface(boolean includeInterfaceView) {
+      this.includeInterfaceView = includeInterfaceView;
+    }
```

Comment:
@stefanobaghino-da Do you think it's worth holding onto this to account for the possibility of more options being added to `InterfaceFilter`?

Reply:
This leaves the room open for future extension without breaking changes, so probably it's a safer choice. If we want to make the life easier to the user without making future extensions break, I would consider the idea of adding a couple of static instances with the boolean set to either `true` or `false`.

Reply:
5073910c411e315bec37309b9eb2007050b8ceb1